### DISCUSSION TITLE
CI integration

### DISCUSSION THREAD

**Discussion by heliocola** [presidentbeef/brakeman#1662] [General]
@presidentbeef : I've recently updated brakeman (5.0.4 -> 5.2.0) in one Rails app and noticed the section about CI in README.MD.
I am using CircleCI and running brakeman for a couple of years, without any problems.
Throughout the years I found 2 different ways to integration, and the most recent is the one I am not using.

I am planning on trying out this 2nd solution, but before that I can document it.
Would it be better to document in the Brakeman Wiki and add a link to it in README.MD?
Or something more like how Jenkis is documented?

**Comment by presidentbeef** [comment]
Probably best to document on the Brakeman wiki and link from the README. That way all the content is in one place.
